openid_connect_configuration.conf

```diff
@@ -8,6 +8,15 @@ map $host $oidc_authz_endpoint {
     #www.example.com "https://my-idp/oauth2/v1/authorize";
 }
 
+map $host $oidc_authz_extra_args {
+    # Extra arguments to be included in the request to the IdP's authorization endpoint.
```

Comment:
As you noticed, we don't always follow this rule, but we try to use the [Code style](http://nginx.org/en/docs/dev/development_guide.html#code_style_general_rules) approach from NGINX dev guide. Try to align the description to 80 characters maximum. Thanks!

Reply:
Thanks for the pointer. I've reworded the description a little and restricted to 80 characters.

Reply:
Thanks @alanwilkie-finocomp!